src/rows.js

```diff
@@ -68,6 +68,24 @@ export class Rows {
     return { value: row, done: false };
   }
 
+  /**
+   * Rows.columns
+   *
+   * Call this if you need column names from the result of a select query.
+   */
+  columns() {
+    if (this._done) {
+      return [];
```

Comment:
I feel it might be more prudent to raise an error in this case, as we don't truly know what the columns are.
```suggestion
      throw new SqliteError("Unable to retrieve column names as transaction is finalised.");
```

(be sure to import `SqliteError`)